src/engine_redraw.c

```diff
@@ -807,7 +807,7 @@ void process_dungeon_top_pointer_graphic(struct PlayerInfo *player)
                 player->flgfield_6 |= PlaF6_Unknown01;
             } else
             {
-                if ((player->field_3 & Pf3F_Unkn02) != 0) {
+                if ((player->field_3 & Pf3F_chosen_subtile_is_not_floor) != 0) {
```

Comment:
gotta love the triple negative

Reply:
yes, there must be a better name, but it does seem to "turn on" under specific circumstances.

Please try to track when it is turned on: `player->field_3 |= Pf3F_chosen_subtile_is_not_floor`
and when it is turned off: `player->field_3 &= ~Pf3F_chosen_subtile_is_not_floor`

to see if you can verify the usage/give a good name :)

Reply:
Resolved in f72161f (renamed to Pf3F_chosen_subtile_is_high)